content/docs/command-reference/pull.md

```diff
@@ -57,11 +57,10 @@ The default remote is used (see `dvc remote default`) unless the `--remote`
 option is used. See `dvc remote` for more information on how to configure a
 remote.
 
-Without arguments, it downloads all files and directories missing from the
-project, found as <abbr>outputs</abbr> of the
-[stages](/doc/command-reference/run) or `.dvc` files present in the workspace.
-The `--all-branches`, `--all-tags`, and `--all-commits` options enable pulling
-multiple Git commits.
+Without arguments, it downloads all files and directories referenced in the
+current workspace (found in `dvc.yaml` and `.dvc` files) that are missing from
+the workspace. The `--all-branches`, `--all-tags`, and `--all-commits` options
+enable pulling multiple Git commits.
```

Comment:
pulling multiple Git commits sounds like we pull actual Git commits

Reply:
Goof point and this also affects fetch, push, and status. Updated them and reordered these sentences a little in 564797d1. PTAL